src/taipy/gui_core/GuiCoreLib.py

```diff
@@ -129,16 +145,19 @@ class _GuiCoreContext(CoreEventConsumerBase):
     __PROP_SCENARIO_PRIMARY = "primary"
     __PROP_SCENARIO_TAGS = "tags"
     __SCENARIO_PROPS = (__PROP_SCENARIO_CONFIG_ID, __PROP_SCENARIO_DATE, __PROP_ENTITY_NAME)
+    __JOB_ACTION = "action"
```

Comment:
That in my opinion is a big vague...
"job_action"?
Really depends on the context where it is used...

Looking at the code below, "action" is fine.
But then the sumbol (__JOB_ACTION) could certainly be renamed to __ACTION and shared